src/panels/lovelace/editor/hui-dialog-edit-view.ts

```diff
@@ -85,8 +85,7 @@ export class HuiDialogEditView extends LitElement {
         .hass="${this.hass}"
         .viewConfig="${this._params.viewConfig}"
         .add="${this._params.add}"
-        @reload-lovelace="${this._params.reloadLovelace}"
-        @cancel-edit-view="${this._cancel}"
```

Comment:
Do you have to remove the registration of these events with typescript?

Reply:
I removed `cancel-edit-view` the other one is still used